### DISCUSSION TITLE
Show another Partition on chromeOS brunch

### DISCUSSION THREAD

**Discussion by mawanx** [sebanc/brunch#868] [Q&A]
HI,is there any guide on how to show storage like in FydeoOS, as i have another storage installed on my laptop. Tanks
![Screenshot 2021-02-10 at 19 22 01](https://user-images.githubusercontent.com/12368558/107593353-c0e77380-6c41-11eb-9616-cbec5f823bec.png)

**Comment by Warrentheo** [comment]
That is covered in the Options section at the bottom of the main page...  Take a look at the "mount_internal_drives" option...
